libs/designsystem/src/lib/components/card/card-header/card-header.component.scss

```diff
@@ -23,3 +36,29 @@ h4 {
   margin: 0;
   font-weight: font-weight('normal');
 }
+
+// TODO: Should 'informational' be added to $notification-colors in general? Would remove the need for mapping to 'semi-light' but could have broader consequences.
+
+$card-flag-colors: (
+  map-merge(
+    $notification-colors,
+    (
+      'informational': get-color('semi-light'),
```

Comment:
The corresponding state on FlagComponent is called `semi-light`, consider aligning (maybe talk with UX @bommariusser )